hudi-common/src/main/java/org/apache/hudi/io/storage/HoodieHFileReader.java

```diff
@@ -223,6 +237,10 @@ public Option getRecordByKey(String key, Schema readerSchema) throws IOException
 
   private R getRecordFromCell(Cell c, Schema writerSchema, Schema readerSchema) throws IOException {
     byte[] value = Arrays.copyOfRange(c.getValueArray(), c.getValueOffset(), c.getValueOffset() + c.getValueLength());
+    return getRecordFromByteValue(value, writerSchema, readerSchema);
+  }
+
+  private R getRecordFromByteValue(byte[] value, Schema writerSchema, Schema readerSchema) throws IOException {
```

Comment:
do you think one line code warrants a separate method? not sure if we really need this. just curious.

Reply:
removed.